From 84baeed49a3d3e0b6b20138cbfe5ecbcfdbc0d67 Mon Sep 17 00:00:00 2001
From: "John B. Wyatt IV" <jwyatt@redhat.com>
Date: Wed, 16 Jul 2025 16:23:27 -0400
Subject: [PATCH 1/4] tuna: Update man page with cpu_power command

[1] added support for the Python libcpupower bindings to control the
cpu power state. This patch documents those changes.

[1]
https://lore.kernel.org/linux-rt-users/20250409193136.44411-1-jwyatt@redhat.com/

Signed-off-by: John B. Wyatt IV <jwyatt@redhat.com>
Signed-off-by: John B. Wyatt IV <sageofredondo@gmail.com>
Signed-off-by: John Kacur <jkacur@redhat.com>
---
 docs/tuna.8 | 18 ++++++++++++++++++
 1 file changed, 18 insertions(+)

diff --git a/docs/tuna.8 b/docs/tuna.8
index 242389455f83..6e30537e9509 100644
--- a/docs/tuna.8
+++ b/docs/tuna.8
@@ -24,6 +24,24 @@ Log application details to file for given LOG-LEVEL
 Print DEBUG level logging details to console
 .SH "COMMANDS"
 .TP
+\fBtuna cpu_power\fR
+usage: tuna-cmd.py cpu_power [-h] (-c CPU-LIST)
+
+Set or query the idle (power) states of all or a selection of cpus in the CPU-LIST. Modifiers \fB\-c\fR may be used to pass the CPU-LIST of cpus to perform the command on.
+
+optional arguments:
+  -h, --help            show this help message and exit
+  -i, --idle-info       Print general idle information for the selected CPUs, including
+                        index values for IDLE-STATE.
+  -s, --status IDLE-STATE
+                        Print whether IDLE-STATE is enabled on the selected CPUs.
+  -d, --disable IDLE-STATE
+                        Disable IDLE-STATE on the selected CPUs.
+  -e, --enable IDLE-STATE
+                        Enable IDLE-STATE on the selected CPUs.
+  -c CPU-LIST, --cpus CPU-LIST
+                        CPU-LIST affected by commands
+.TP
 \fBtuna isolate\fR
 usage: tuna-cmd.py isolate [-h] (-c CPU-LIST | -S CPU-SOCKET-LIST | -N)
 
-- 
2.49.0

